tools/linter_lib/custom_check.py

```diff
@@ -446,23 +446,39 @@ def build_custom_checkers(by_lang):
     ] + whitespace_rules[0:1]  # type: RuleList
     css_rules = cast(RuleList, [
         {'pattern': '^[^:]*:\S[^:]*;$',
-         'description': "Missing whitespace after : in CSS"},
+         'description': "Missing whitespace after : in CSS",
+         'good_lines': ["background-color: white;", "text-size: 16px;"],
+         'bad_lines': ["background-color:white;", "text-size:16px;"]},
         {'pattern': '[a-z]{',
-         'description': "Missing whitespace before '{' in CSS."},
+         'description': "Missing whitespace before '{' in CSS.",
+         'good_lines': ["input {", "body {"],
+         'bad_lines': ["input{", "body{"]},
         {'pattern': 'https://',
-         'description': "Zulip CSS should have no dependencies on external resources"},
+         'description': "Zulip CSS should have no dependencies on external resources",
+         'good_lines': ["background: url(\"background.jpg\")"],
```

Comment:
This is a better good example (from actual code):
`url(/static/images/landing-page/pycon.jpg);`